platforms/ice40_up5k_b_evn.py

```diff
@@ -31,13 +31,13 @@
 
 # The ICE40UP5K-B-EVN does not use the provided FT2232H chip to provide a
 # UART port. One must use their own USB-to-serial cable instead to get a UART.
-# We have chosen to use 37A and 36B for "tx" and "rx" respectively on Header B
+# We have chosen to use 48B and 51A for "tx" and "rx" respectively on Header B
 # to implement UART connections. The board comes unpopulated and will need to
 # have headers soldered.
```

Comment:
Why the change here?

Reply:
Pins 37A and 37B are shared with switches on this board, so I have to choose either one functionality or the other.